src/device-registry/utils/generate-filter.js

```diff
@@ -747,6 +747,7 @@ const generateFilter = {
       airqloud_id,
```

Comment:
With the new query fields (`device_codes`, `activity_codes`, etc) in place, making the relevant updates to our filter util